src/ngraph_builder.cc

```diff
@@ -857,6 +857,49 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
       ng_op_map[op->name()] = ng_expand_dim;
     }
     // --------
+    // Fill
+    // --------
+    else if (op->type_string() == "Fill") {
+
+      tf::Node* tf_dims;
+      tf::Node* tf_value;
+      TF_RETURN_IF_ERROR(op->input_node(0, &tf_dims));
+      TF_RETURN_IF_ERROR(op->input_node(1, &tf_value));
+
+      shared_ptr<ng::Node> ng_dims;
+      try {
+        ng_dims = ng_op_map.at(tf_dims->name()); 
+      } catch (const std::out_of_range&) {
+        return tf::errors::InvalidArgument("Missing input: " + tf_dims->name());
+      }
+
+      shared_ptr<ng::Node> ng_value;
+      try {
+        ng_value = ng_op_map.at(tf_value->name()); 
+      } catch (const std::out_of_range&) {
+        return tf::errors::InvalidArgument("Missing input: " + tf_value->name());
+      }
+      ng_value = ng_op_map.at(tf_value->name());
+     
+      auto ng_dims_const =
+          std::dynamic_pointer_cast<ng::op::Constant>(ng_dims);
```

Comment:
Instead of trying to force the argument to `ng::op::Constant` here, we should use the value in the `_ngraph_fill_static_dims` field. (Under some circumstances, the constant node can end up being placed in a separate graph.)